inc/mansidebar.shtml

```diff
@@ -5,6 +5,7 @@
     <ul>
       <li><a href="/docs/manmaster">master</a></li>
       <li><a href="/docs/man1.1.0">1.1.0</a></li>
+      <li><a href="/docs/man1.1.1">1.1.1</a></li>
```

Comment:
shouldn't this be before 1.1.0 ?

Reply:
Oh, sure. That was my original intention :-). I will fix it.

Reply:
Done.